build/xunit.runsettings

```diff
@@ -10,6 +10,9 @@
         <DataCollectors>
             <DataCollector friendlyName="blame" enabled="True">
                 <Configuration>
+                    <!-- Enables crash dump-->
+                    <CollectDump DumpType="Full" />
+                    <!-- Enables hang dump-->
```

Comment:
nit: I don't really get what the comments are adding as the config itself is very descriptive.

Reply:
I'll remove it later in another PR.